Fonts/PTSenif-Bold/specimen_files/specimen_stylesheet.css

```diff
@@ -0,0 +1,492 @@
+@import url('grid_12-825-55-15.css');
+
+/*
+	CSS Reset by Eric Meyer - Released under Public Domain
+    http://meyerweb.com/eric/tools/css/reset/
+*/
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+font,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
+  font-size: 100%;
+  vertical-align: baseline;
+  background: transparent;
+}
+
+body {
+  line-height: 1;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+
+blockquote,
+q {
+  quotes: none;
+}
+
+q::after {
+  content: '';
+  content: none;
+}
+
+:focus {
+  outline: 0;
+}
+
+ins {
+  text-decoration: none;
+}
+
+del {
+  text-decoration: line-through;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+body {
+  color: white;
+  background-color: #dcdcdc;
+}
+
+a {
+  text-decoration: none;
+  color: #1883ba;
+}
+
+h1 {
+  font-size: 32px;
+  font-weight: normal;
+  font-style: normal;
+  margin-bottom: 18px;
+}
+
+h2 {
+  font-size: 18px;
+}
+
+#container {
+  width: 865px;
+  margin: auto;
+}
+
+#header {
+  padding: 20px;
+  font-size: 36px;
+  background-color: #000;
+  color: #fff;
+}
+
+#header span { color: #666; }
+
+#main_content {
+  background-color: #fff;
+  padding: 60px 20px 20px;
+}
+
+#footer
+p {
+  margin: 0;
+  padding-top: 10px;
+  padding-bottom: 50px;
+  color: #333;
+  font: 10px Arial, sans-serif;
+}
+
+.tabs {
+  width: 100%;
+  height: 31px;
+  background-color: #444;
+}
+
+.tabs li {
+  float: left;
+  margin: 0;
+  overflow: hidden;
+  background-color: #444;
+}
+
+.tabs
+li a {
+  display: block;
+  color: #fff;
+  text-decoration: none;
+  font: bold 11px/11px;
+  text-transform: uppercase;
+  padding: 10px 15px;
+  border-right: 1px solid #fff;
+}
+
+.tabs
+li a:hover {
+  background-color: #00b3ff;
+}
+
+.tabs li {
+  color: weight;
+  background-color: #fff;
+}
+
+.active
+a {
+  color: weight;
+  background-color: #fff;
+}
+
+div.huge {
+  font-size: 300px;
+  line-height: 1em;
+  padding: 0;
+  letter-spacing: -.02em;
+  overflow: hidden;
+}
+
+div.glyph_range {
+  font-size: 72px;
+  line-height: 1.1em;
+}
+
+.size10 { font-size: 10px; }
+.size11 { font-size: 11px; }
+.size12 { font-size: 12px; }
+.size13 { font-size: 13px; }
+.size14 { font-size: 14px; }
+.size16 { font-size: 16px; }
+.size18 { font-size: 18px; }
+.size20 { font-size: 20px; }
+.size24 { font-size: 24px; }
+.size30 { font-size: 30px; }
+.size36 { font-size: 36px; }
+.size48 { font-size: 48px; }
+.size60 { font-size: 60px; }
+.size72 { font-size: 72px; }
+.size90 { font-size: 90px; }
+
+.psample_row1 { height: 120px;}
+
+.psample_row1 { height: 120px;}
+
+.psample_row2 { height: 160px;}
```

Comment:
Codacy found an issue: [Expected single space before "}" of a single-line block (block-closing-brace-space-before)](https://app.codacy.com/gh/WebHeroSchool/-Kostikova-Irina/pullRequest?prid=6241956)